spec/simple_writer_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe SimpleWriter do
```

Comment:
The `SimpleWriter` class does not increase its responsibility. Indeed it is untouched even though the output needs more variation  and its combination. As you can see, we closed the class for modification, but it keeps opened for extension. The others writers are its extension.